src/odemis/gui/cont/correlation.py

```diff
@@ -238,14 +238,22 @@ def on_char(self, evt: wx.Event) -> None:
         """handle key presses
         :param evt: (wx.Event) the event"""
 
-        if not self.correlation_enabled():
-            logging.debug("correlation not enabled, passing event to canvas")
-            active_canvas = evt.GetEventObject()
-            active_canvas.on_char(evt)            # super event passthrough
-
+        # event data
         key = evt.GetKeyCode()
         shift_mod = evt.ShiftDown()
 
+        # pass event to canvas, if not a valid correlation key
+        valid_keys = [wx.WXK_LEFT, wx.WXK_RIGHT, wx.WXK_UP, wx.WXK_DOWN]
+        if evt.GetKeyCode() not in valid_keys:
+            logging.debug(f"invalid correlation key, passing event to canvas, key was: {key}")
+            active_canvas = evt.GetEventObject()
+            active_canvas.on_char(evt)
+            return
+
+        if not self.correlation_enabled():
```

Comment:
So in this tab, the arrow keys will never move the canvas the standard way? Either they move a stream, or they have no effect. Is that the intention? If so, can you add a comment to clarify.

Reply:
You're right, I've moved this into the above check and done the passthrough with evt.Skip